### ISSUE TITLE
Extra specifications needed for local install of neper

### ISSUE THREAD

**Issue by rcarson3** [rquey/neper#13]
I found that I add to add the following lines within the top-level ```CMakeLists.txt``` file in order to get things to build when I had to use a local install of NLOPT and neper due to a lack of admin/sudo access on this machine.

```cmake
if (HAVE_NLOPT)                                     # NLOPT
  find_package(NLOPT REQUIRED)                      # NLOPT
  set (EXTRA_LIBS ${EXTRA_LIBS} ${NLOPT_LIBRARIES}) # NLOPT
  include_directories (${NLOPT_INCLUDE_DIRS})       # NLOPT
endif ()
```

I also had to specify the following flags: ```-DNLOPT_LIBRARY``` and ```-DNLOPT_INCLUDE_DIR``` on the command line when I first ran cmake in order to point everything in the right directions.

**Comment by rcarson3** [comment]
@rquey  I also decided to test this on a MacOS system where GSL was installed locally rather than globally. I did turn off the OpenMP option for this case but that's just because Apple still doesn't include it installed by default... It required that I also add the following lines to the top level ```CMakeLists.txt``` file in order to get things to build.

```cmake
if (HAVE_GSL)
  find_package(GSL REQUIRED)
  set (EXTRA_LIBS ${EXTRA_LIBS} ${GSL_LIBRARIES})
  include_directories (${GSL_INCLUDE_DIRS})
endif ()
```

I also had to specify the following flag ```-DGSL_CONFIG``` on the command line when I first ran cmake in order to point everything in the right direction. I also found that this flag required you to point to directly located to the config file located in ```$GSL_INSTALL_DIR/bin/```.  Afterwards, everything installed just fine.
